source/vscode/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# What's New in QDK
+
+## v1.2.3
+
+- Added feature X
+- Improved Y
+- Fixed bug Z
```

Comment:
We can't merge it with a mock changelog, as this is what will automatically show up for the extension as soon as the next build is published. Can you add the contents for the past couple of releases? You can use the releases page on GitHub for reference.

Reply:
I've filled in this changelog with the changelogs of 1.18.0,, 1.17.0, and 1.16.0.